### PR TITLE
8265107: Fix effect problems

### PR DESCRIPTION
**Pull request by Wanghuang-Huawei**
* Fix some effect problems

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Change must be properly reviewed

### Issue
 * [JDK-8265107](https://bugs.openjdk.java.net/browse/JDK-8265107): Fix effect problems

### Reviewers
 * [Xiaohong Gong](https://openjdk.java.net/census#xgong) (@XiaohongGong - Committer)

### Contributors
 * Ai Jiaming `<aijiaming1@huawei.com>`
 * Wang Huang `<whuang@openjdk.org>`

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/panama-vector pull/66/head:pull/66` \
`$ git checkout pull/66`

Update a local copy of the PR: \
`$ git checkout pull/66` \
`$ git pull https://git.openjdk.java.net/panama-vector pull/66/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 66`

View PR using the GUI difftool: \
`$ git pr show -t 66`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/panama-vector/pull/66.diff">https://git.openjdk.java.net/panama-vector/pull/66.diff</a>

</details>

### CODE REVIEW

File: src/hotspot/cpu/aarch64/aarch64_sve_ad.m4 [hunk 1]
```diff
@@ -2150,27 +2148,24 @@ VTEST_PARTIAL(anytrue, ne,      -1, NE)
...
   match(Set dst (VectorInsert (Binary src val) idx));
-  effect(TEMP tmp, TEMP pTmp, KILL cr);
+  effect(TEMP_DEF dst, TEMP pTmp, KILL cr);
   ins_cost(4 * SVE_COST);
-  format %{ "sve_index $tmp, $3, -16, 1\n\t"
-            "sve_cmpeq $pTmp, $tmp, ($idx-#16) // shift from [0, 31] to [-16, 15]\n\t"
+  format %{ "sve_index $dst, $3, -16, 1\n\t"
+            "sve_cmpeq $pTmp, $dst, ($idx-#16) // shift from [0, 31] to [-16, 15]\n\t"
             "sve_orr $dst, $src, $src\n\t"
             "sve_cpy $dst, $pTmp, $val\n\t# insert into vector ($1)" %}
```
Comment (nsjian) [inline src/hotspot/cpu/aarch64/aarch64_sve_ad.m4:2161]: \n\t -> \t for the last line of the block.

Comment (Wanghuang-Huawei) [reply src/hotspot/cpu/aarch64/aarch64_sve_ad.m4]: Thank you for your review. I have fixed this format problem.

File: src/hotspot/cpu/aarch64/aarch64_sve.ad [hunk 2]
```diff
@@ -3542,111 +3542,99 @@ instruct vtest_anytrue_partial(iRegINoSp dst, vReg src1, vReg src2, pRegGov pTmp
...
-             as_FloatRegister($src$$reg));
-    }
+                 as_FloatRegister($dst$$reg), (int)($idx$$constant) - 16);
+    __ sve_orr(as_FloatRegister($dst$$reg),
+               as_FloatRegister($src$$reg),
+               as_FloatRegister($src$$reg));
     __ sve_cpy(as_FloatRegister($dst$$reg), __ B,
                as_PRegister($pTmp$$reg), as_Register($val$$reg));
 
   %}
```
Comment (XiaohongGong) [inline src/hotspot/cpu/aarch64/aarch64_sve.ad:3566]: Remove the extra line (line 3565)?

Comment (Wanghuang-Huawei) [reply src/hotspot/cpu/aarch64/aarch64_sve.ad]: Sure. I have resolved that. Thank you.

**Review by XiaohongGong — APPROVED** [review]

**Comment by XiaohongGong** [comment]
LGTM, thanks!

**Pull request closed without merging**